tests/unit/App.spec.ts

```diff
@@ -882,8 +886,8 @@ describe('App as a Draft IA with approved NR', () => {
 
   beforeAll(() => {
     // clear store
-    store.commit('tasks', [])
-    store.commit('mutateFilings', [])
+    rootStore.setTasks([])
+    filingHistoryListStore.mutateFilings([])
```

Comment:
Oh, you didn't rename mutateFilings -> setFilings ?

Reply:
I see setFilings() calls mutateFilings(). Otherwise it's just the unit tests using mutateFilings(). Could the unit tests call setFilings() instead (and delete mutateFilings())?